src/components/atoms/VisusalisationsSelector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
```VisusalisationsSelector.tsx```
Not to be Mr. Grammarly or anything, but I think there is something fishy🐟 about the filename. I suggest changing it to VisualisationSelector.